specs/core/0_beacon-chain.md

```diff
@@ -878,8 +878,9 @@ def get_committee_count_per_slot(active_validator_count: int) -> int:
 
 ```python
 def get_shuffling(randao_mix: Hash32,
-                  validators: List[ValidatorRecord],
-                  slot: int) -> List[List[int]]
+                  validators: List[Validator],
+                  crosslinking_start_shard: int,
```

Comment:
I think this `crosslinking_start_shard` got added in a bad merge or something. Remove this line